smokefree/sites/all/modules/custom/icfi_bqp/bqp-template.tpl.php

```diff
@@ -1,3 +1,17 @@
+<?php
+
+// create the og:description tag
+$header = array(
+'#tag' => 'meta',
+'#attributes' => array(
+  'property' => 'og:description',
+  'content' => 'Prepare to quit with a personalized quit plan. Build your quit plan in 7 easy steps.',
+  ),
+);
+drupal_add_html_head($header, 'bpq_description');
+
+?>
+
```

Comment:
We should stick to presentation in the template file and put PHP business logic in a preprocessing function. I think you can create a function template_preprocess_bqp_theme in the .module file and add this code there.

Reply:
I added this to the template file because this adds a meta tag to the HTML page. No business logic has been added to the code here, it is all modifications to the presentation layer.